ui/app/src/components/Widget/Widget.tsx

```diff
@@ -51,6 +54,8 @@ const Widget = memo(function(props: WidgetProps) {
     } else {
       return <NonReactWidget widget={record} configuration={configuration} />;
     }
+  } else if (!plugin) {
+    return <EmptyState title={formatMessage(messages.componentNotFound)} styles={{ image: { width: 100 } }} />;
```

Comment:
Add component id to message. Message should end up as: 
empty state title: Component "{id}" not found.
empty state subtitle: Check ui config & make sure you've installed the plugins that contain the desired components.